### ISSUE TITLE
Can't bypass NHentai Cloudflare, other way to bypass cloudflare with HDoujin?

### ISSUE THREAD

**Issue by Sky49081** [HDoujinDownloader/HDoujinDownloader#314]
Flaresolverr has not been updated to bypass the NHentai Cloudflare, are there any other way to bypass the cloudflare with HDoujin?

**Comment by HDoujinDownloader** [comment]
You're right, FlareSolverr hasn't been working lately and has yet to receive an update. There is work happening on this, but it might be a while before it's merged in: FlareSolverr/FlareSolverr#1163

In the meantime, you can still bypass Cloudflare manually by following the instructions here: https://github.com/HDoujinDownloader/HDoujinDownloader/wiki/Cloudflare#bypassing-cloudflare-manually

Also, I'm currently working on another feature that will help with this. I'm adding an integrated web browser to HDoujin Downloader, which will help with Cloudflare and captchas.
